llama_stack/templates/remote-vllm/doc_template.md

```diff
@@ -85,11 +85,14 @@ export INFERENCE_PORT=8000
 export INFERENCE_MODEL=meta-llama/Llama-3.2-3B-Instruct
 export LLAMA_STACK_PORT=8321
 
+# You need a local checkout of llama-stack to run this, get it using
```

Comment:
fyi, wording is copied from the existing example in this very document for safety stack:
https://github.com/meta-llama/llama-stack/blob/193e53121650700b4e3b435a4f1b12119b761386/llama_stack/templates/remote-vllm/doc_template.md?plain=1#L106-L107